docs/sql-guide/sql-guide-home.md

```diff
@@ -18,7 +18,7 @@ FeatureBase supports different types of queries that are discussed further in th
 {% include /cloud/cloud-db-create-before-begin.md %}
 
 {: .note}
-FeatureBase Community can run SQL through the FeatureBase CLI by [Enabling the SQL endpoint at startup](/docs/community/com-startup-connect)
+FeatureBase Community can run SQL through the commandline interactive tool [fbsql](/docs/community/com-startup-connect) or through the lattice Web UI.
```

Comment:
This sentence has grammatical errors:
* "FeatureBase Community can run SQL" > FeatureBase Community runs SQL at the back-end.  Only users can use the FBSQL tool
* commandline -- this should be two words
* "interactive tool" is what's called a "tautology" which means you're saying the same thing twice -- in this case, a tool is implied to be interactive by its very existence. Also "interactive" is over-used and ultimately meaningless in all but a very few use cases. So don't use it :)
* You have two ways to run SQL here, both of which should be hyperlinks. And in fact there are 3 ways -- the query tool, the query builder, the **other** command-line SQL tool, and fbsql
* I've yet to get a clear explanation of what the "lattice Web UI" actually is. If you're talking about the Community GUI, it's best to just say that. 

An alternative approach: 

```
## Before you begin
{% include /cloud/cloud-db-create-before-begin.md %}

* [Learn how to query FeatureBase databases using terminal commands](/docs/tools/tool-fbsql)
* [Learn how to query FeatureBase databsaes using the SQL Command Line Interface](/docs/community/com-config/com-config-sql-cli-enable) <-- filename depends on existing PR "APPROVE ME FIRST! > Formatting fixes
* Login to FeatureBase Community and run queries using the Query editor or Query builder

```